app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,6 @@
+<ul>
```

Comment:
I'd expect this template to reflect what we currently show in the homepage (using this template instead)